### DISCUSSION TITLE
Status Code 410 na Consulta de Payloads de QR Code dinâmico (imediato e com vencimento)

### DISCUSSION THREAD

**Discussion by mzavarize** [bacen/pix-api#375] [Q&A]
Bom dia, 

No manual para padrões de Iniciação do Pix versão 2.3, no item 1.6.5 Sobre Cobranças Concluídas informa que pode retornar um Status Code 410, esse status code poderá ser retornado também para os status “REMOVIDA_PELO_USUARIO_RECEBEDOR”,
“REMOVIDA_PELO_PSP”?

Atencisamente,

**Comment by ninrod** [comment]
bom dia @mzavarize , 

Não. 410 simplesmente indica que um recurso que existia na URL em questão não existe mais.

**Reply by mzavarize** [reply]
Obrigado
